db/models.py

```diff
@@ -46,3 +48,62 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
+    pass
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+
+    class Meta:
+        ordering = ["-user"]
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(
+        to=MovieSession, on_delete=models.CASCADE
+    )
+    order = models.ForeignKey(to=Order, on_delete=models.CASCADE)
+    row = models.PositiveSmallIntegerField()
+    seat = models.PositiveSmallIntegerField()
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["row", "seat", "movie_session"],
+                name="row_seat_movie_session"
+            )
+        ]
+
+    def clean(self) -> None:
+        c_hall_rows = self.movie_session.cinema_hall.rows
+        c_hall_seat = self.movie_session.cinema_hall.seats_in_row
```

Comment:
u can just create variable for self.movie_session.cinema_hall and than call rows/seats etc. no need to create variables for attributes